### PR TITLE
New version: Metanorma.Metanorma version 1.11.1

### PR DESCRIPTION
**Pull request by CAMOBAP**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/192060)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-11-17 01:41:30.149 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Metanorma.Metanorma.1.11.1\extracted\metanorma.exe
> MSI (s) (E0:40) [01:41:10:282]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (E0:40) [01:41:10:282]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (E0:40) [01:41:10:282]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (E0:40) [01:41:10:282]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (E0:40) [01:41:10:282]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.
> Property(S): OutOfDiskSpace = 0
> Property(S): OutOfNoRbDiskSpace = 0
> Property(S): PrimaryVolumeSpaceAvailable = 0
> Property(S): PrimaryVolumeSpaceRequired = 0
> Property(S): PrimaryVolumeSpaceRemaining = 0
> MSI (s) (E0:B8) [01:41:11:172]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (E0:B8) [01:41:11:172]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (E0:B8) [01:41:11:172]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (E0:B8) [01:41:11:172]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (E0:B8) [01:41:11:172]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.

Automated error analysis suggests -1978335141 may mean Unable to locate nested installer. Does the `RelativeFilePath` have the previous version number in it?

(Automated response - build 897.)

**Comment by CAMOBAP** [comment]
Hmm, but the `metanorma.exe` is not installed it's ready to use executable file.

This upload just a new version, previous package had the same zip with `metanorma.exe` in it...

Let me have a look maybe something has changed

File: manifests/m/Metanorma/Metanorma/1.11.1/Metanorma.Metanorma.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Metanorma.Metanorma
+PackageVersion: 1.11.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: metanorma.exe
```
Comment (stephengillie) [inline manifests/m/Metanorma/Metanorma/1.11.1/Metanorma.Metanorma.installer.yaml:9]:
```suggestion
- RelativeFilePath: metanorma-windows-x86_64.exe
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @CAMOBAP,

The change appears to be in the filename:
image

File: manifests/m/Metanorma/Metanorma/1.11.1/Metanorma.Metanorma.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  PortableCommandAlias: metanorma
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/metanorma/packed-mn/releases/download/v1.11.1/metanorma-windows-x86_64.zip
+  InstallerSha256: 1FBC2E2C3542E89AFAE17FAF87A641CE571B46E65AE5298F6852F5D1CDBE873A
```
Comment (CAMOBAP) [inline manifests/m/Metanorma/Metanorma/1.11.1/Metanorma.Metanorma.installer.yaml:15]:
```suggestion
  InstallerSha256: 7C8EFCBBB3B21D37DD413F2B731306E41C357E9FBB6E029A6441AE6C418E4B4F
```

File: manifests/m/Metanorma/Metanorma/1.11.1/Metanorma.Metanorma.installer.yaml (hunk 1 above)
Comment (CAMOBAP) [reply manifests/m/Metanorma/Metanorma/1.11.1/Metanorma.Metanorma.installer.yaml:9]: @stephengillie thanks for spotting this, and sorry for the inconvenience there was an issue with the package itself.

Just reuploaded the package

Is it possible to rerun the ci?

Comment (CAMOBAP) [reply manifests/m/Metanorma/Metanorma/1.11.1/Metanorma.Metanorma.installer.yaml:9]: upd, looks like it's in progress already after update of the sha256

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-11-18 14:17:09.661 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Metanorma.Metanorma.1.11.1\extracted\metanorma.exe
> MSI (s) (98:3C) [14:16:40:600]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (98:3C) [14:16:40:600]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (98:3C) [14:16:40:600]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (98:3C) [14:16:40:600]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (98:3C) [14:16:40:600]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.
> Property(S): OutOfDiskSpace = 0
> Property(S): OutOfNoRbDiskSpace = 0
> Property(S): PrimaryVolumeSpaceAvailable = 0
> Property(S): PrimaryVolumeSpaceRequired = 0
> Property(S): PrimaryVolumeSpaceRemaining = 0
> MSI (s) (98:6C) [14:16:41:444]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (98:6C) [14:16:41:444]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (98:6C) [14:16:41:444]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (98:6C) [14:16:41:444]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (98:6C) [14:16:41:444]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.

Automated error analysis suggests -1978335141 may mean Unable to locate nested installer. Does the `RelativeFilePath` have the previous version number in it?

(Automated response - build 898.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-11-19 12:16:21.464 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Metanorma.Metanorma.1.11.1\extracted\metanorma.exe
> MSI (s) (3C:C8) [12:15:48:369]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (3C:C8) [12:15:48:369]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (3C:C8) [12:15:48:369]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (3C:C8) [12:15:48:369]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (3C:C8) [12:15:48:369]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.
> Property(S): OutOfDiskSpace = 0
> Property(S): OutOfNoRbDiskSpace = 0
> Property(S): PrimaryVolumeSpaceAvailable = 0
> Property(S): PrimaryVolumeSpaceRequired = 0
> Property(S): PrimaryVolumeSpaceRemaining = 0
> MSI (s) (3C:78) [12:15:49:228]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (3C:78) [12:15:49:228]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (3C:78) [12:15:49:228]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (3C:78) [12:15:49:228]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (3C:78) [12:15:49:228]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.

Automated error analysis suggests -1978335141 may mean Unable to locate nested installer. Does the `RelativeFilePath` have the previous version number in it?

(Automated response - build 898.)

**Pull request closed without merging**